content/business-models/none.md

```diff
@@ -4,16 +4,16 @@ date: 2019-01-09
 weight: 1
 ---
 
-This is typically the ur-form of sustainable Free and Open Source Communities.
+This is typically the starting point for Sustainable Free and Open Source Communities.
```

Comment:
"Starting point" sounds like you're supposed to leave this point and go on. While in reality most projects do not and don't have to be.

How about **archetype**?

> This is typically the archetype of Sustainable

Reply:
If we're moving further away from ur-from. Then maybe "common" instead of typical as well?

Reply:
typically, commonly. [you say tomatoes I say tomatos](https://www.youtube.com/watch?v=q7a4z7zLKts&feature=youtu.be&t=59) :)